index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main-grid">
+        <section class="navigation">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/X_logo_2023_%28white%29.png/900px-X_logo_2023_%28white%29.png">
+        <h4>Home</h4>
+        <h4>Explore</h4>
+        <h4>Notifications</h4>
+        <h4>Messages</h4>
+        <h4>Bookmarks</h4>
+        <h4>Profile</h4>
+        <h4>More</h4>
+        <h3>Post</h3>
+        </section>
+
+        <section class="middle">
+            <h1>Home</h1>
+            <section class="middle-top">
+                <img src="https://newprofilepic.photo-cdn.net//assets/images/article/profile.jpg?90af0c8">
+            <input type="text" placeholder="What's happening?">
```

Comment:
Please can you explain this? It looks like you've added some interactivity to your file maybe